packages/flutter/lib/src/widgets/routes.dart

```diff
@@ -113,7 +119,7 @@ abstract class TransitionRoute<T> extends OverlayRoute<T> {
 
   T _result;
 
-  void handleStatusChanged(AnimationStatus status) {
+  void _handleStatusChanged(AnimationStatus status) {
```

Comment:
looks like I made it public in https://github.com/flutter/flutter/commit/88e74693ae36942503ea42a17ea873725f544f55